VERSIONING.md

```diff
@@ -0,0 +1,24 @@
+# Versioning in fundamental-ngx
```

Comment:
This can be shortened and added to [`CONTRIBUTING.md`](https://github.com/SAP/fundamental-ngx/blob/main/CONTRIBUTING.md)